axelor-human-resource/src/main/resources/i18n/messages_fr.csv

```diff
@@ -951,8 +951,8 @@
 "Refused Timesheet Template","Modèle pour feuille de temps refusée",,
 "Register payment","Enregistrer un paiement",,
 "Registration certificate","Certificat d'immatriculation",,
-"Registration code","N° siret",,
-"Registration number","Matricule",,
+"Registration code",,,
+"Registration number",,,

```

Comment:
please check with @jmi-axelor what could replace these translations in french. Even if we do not see in the form view the titles, there still will be part of the application where the fields title will appear, so we need a translation.

Reply:
Registration number is already translated in axelor-base